### PR TITLE
W-14901181 fins 1.10 beta to latest

### PR DESCRIPTION
**Pull request by tasmoore**

### CODE REVIEW

**Review by Crispy-Salesforce — DISMISSED** [review]

**Review by bjoson-mulesoft — APPROVED** [review]

File: financial-services/1.10/modules/ROOT/pages/fins-assets.adoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+include::partial$listeners-fragment.adoc[]
+
+== FINS Custom Components
+
+include::partial$custom-components-fragment.adoc[]
+
+== Accelerator Shared APIs
+
+The following shared APIs are are leveraged by this accelerator.
```
Comment (bjoson-mulesoft) [inline financial-services/1.10/modules/ROOT/pages/fins-assets.adoc:27]:
```suggestion
The following shared APIs are leveraged by this accelerator.
```

File: financial-services/1.10/modules/ROOT/pages/prerequisites.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
+= Prerequisites for MuleSoft Accelerator for Financial Services
+
+This page provides a summary of the guides available for configuring the various systems and products required to implement the use cases provided by this solution. The individual systems needed, and amount of configuration required, depend on which assets are to be deployed in your target environment(s) - refer to the use case pages for more details.
```
Comment (bjoson-mulesoft) [inline financial-services/1.10/modules/ROOT/pages/prerequisites.adoc:3]: Should this be an m-dash?

File: financial-services/1.10/modules/ROOT/pages/upgrade-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+. Apply the latest configuration changes to your Salesforce Financial Services Cloud instance as documented in the customization guide.
+. Deploy the latest ID Graph LWC asset to your target Salesforce org(s).
+. Apply the latest schema changes to the Core Banking and MDM databases.
+** This may require some updates to existing records.
+. Deploy the latest customer sync components included in this release.
+
+=== Verification Steps
+
+* Create a new customer in Salesforce FSC, ensure it is replicated to Core Banking and MDM.
```
Comment (bjoson-mulesoft) [inline financial-services/1.10/modules/ROOT/pages/upgrade-notes.adoc:69]: This isn't a complete sentence as written. Either "To create ..." or "FSC; ensure"

File: financial-services/1.10/modules/ROOT/pages/upgrade-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+* Update the same customer in FSC, ensure the update is reflected in Core Banking and MDM.
```
Comment (bjoson-mulesoft) [inline financial-services/1.10/modules/ROOT/pages/upgrade-notes.adoc:70]: Same as the above

File: financial-services/1.10/modules/ROOT/pages/upgrade-notes.adoc [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+== Release 1.2 to Release 1.3
+
+In addition to new components introduced for the Payments Modernization use case, this release also includes updates to existing Core Banking, MDM, and Salesforce integration components.
+
+=== Change Summary
+
+The following changes may have an impact on existing deployments:
+
+* Objects in the Banking Library and associated database schemas have been expanded.
```
Comment (bjoson-mulesoft) [inline financial-services/1.10/modules/ROOT/pages/upgrade-notes.adoc:84]:
```suggestion
* Objects in the Banking Library and associated database schemas are expanded.
```

File: financial-services/1.10/modules/ROOT/pages/upgrade-notes.adoc [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+* Additional support for Organization parties in the customer sync process was introduced.
```
Comment (bjoson-mulesoft) [inline financial-services/1.10/modules/ROOT/pages/upgrade-notes.adoc:85]:
```suggestion
* Additional support for Organization parties in the customer sync process is introduced.
```

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**